armory/datasets/preprocessing.py

```diff
@@ -57,6 +58,54 @@ def carla_over_obj_det_dev(element, modality="rgb"):
     )
 
 
+def carla_video_tracking_preprocess(element, max_frames, split):
+    x = element["video"]  # tf.Tensor [B, H, W, C]
+    y = element["bboxes"]  # tf.Tensor [B, 4]
+    y_meta = element["patch_metadata"]  # Dict
```

Comment:
for consistency with the rest of the codebase, I'd suggest renaming `y_meta` to `y_patch_metadata`

Reply:
Addressed in 8db643f627562c3b68265725f1a7bee85d06ada6.